validator/client/src/main/java/tech/pegasys/teku/validator/client/signer/ExternalSigner.java

```diff
@@ -175,12 +187,18 @@ private String createSigningRequestBody(
   }
 
   private BLSSignature getBlsSignature(
-      final HttpResponse<String> response, final Throwable throwable) {
+      final HttpResponse<String> response,
+      final Throwable throwable,
+      final Supplier<String> slashableMessage) {
     if (throwable != null) {
       throw new ExternalSignerException(
           "External signer failed to sign due to " + throwable.getMessage(), throwable);
     }
 
+    if (response.statusCode() == 412) {
```

Comment:
Would be good to sue a constant here (and actually switch the 200 below to a `HttpStatusCodes` constant too.